pythnet/message_buffer/programs/message_buffer/src/instructions/resize_buffer.rs

```diff
@@ -47,15 +55,16 @@ pub struct ResizeBuffer<'info> {
     pub system_program: Program<'info, System>,
```

Comment:
I think you missed some places in replacing "message" by "MESSAGE"

Reply:
fixed thanks